### PR TITLE
Describe the HtmlEditor - Tables demo

### PR DESCRIPTION
**Pull request by AlexJameson**
http://jsserver:81/Demos-20.2/WidgetsGallery/Demo/HtmlEditor/Tables/jQuery/Light/

image

[DP](https://github.com/DevExpress/DevExtreme/issues/14724)
[Code](https://github.com/DevExpress/devextreme-demos/pull/155/files#diff-da5ea41d8d7065f2b81dbd45c6fc552990f4be7a174e4d40f0ab660512872a97)

### CODE REVIEW

File: JSDemos/Demos/HtmlEditor/Tables/description.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+DevExtreme **HtmlEditor** allows you to create and manage tables. This demo shows how to perform the following operations in the UI:
```
Comment (RomanTsukanov) [inline JSDemos/Demos/HtmlEditor/Tables/description.md]: > allows you

Or users?

> This demo shows how to perform the following operations in the UI:

I suspect that these are toolbar items, not operations. So, maybe it is better to focus on how to enable the table management capabilities? Something like "To enable this functionality, add the following items to the toolbar:"

File: JSDemos/Demos/HtmlEditor/Tables/description.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#inserttable" target="_blank">insertTable</a>
+- <a href="" target="_blank">insertRowAbove</a> / <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#insertrowbelow" target="_blank">insertRowBelow</a>
```
Comment (RomanTsukanov) [inline JSDemos/Demos/HtmlEditor/Tables/description.md]: `insertRowAbove` misses a link

File: JSDemos/Demos/HtmlEditor/Tables/description.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#insertcolumnright" target="_blank">insertColumnRight / <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#insertcolumnleft" target="_blank">insertColumnLeft
```
Comment (RomanTsukanov) [inline JSDemos/Demos/HtmlEditor/Tables/description.md]:
```suggestion
- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#insertcolumnright" target="_blank">insertColumnRight</a> / <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#insertcolumnleft" target="_blank">insertColumnLeft</a>
```

File: JSDemos/Demos/HtmlEditor/Tables/description.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deletetable" target="_blank">deleteTable
```
Comment (RomanTsukanov) [inline JSDemos/Demos/HtmlEditor/Tables/description.md]:
```suggestion
- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deletetable" target="_blank">deleteTable</a>
```

File: JSDemos/Demos/HtmlEditor/Tables/description.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deleterow" target="_blank">deleteRow / <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deletecolumn" target="_blank">deleteColumn
```
Comment (RomanTsukanov) [inline JSDemos/Demos/HtmlEditor/Tables/description.md]:
```suggestion
- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deleterow" target="_blank">deleteRow</a> / <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deletecolumn" target="_blank">deleteColumn</a>
```

File: JSDemos/Demos/HtmlEditor/Tables/description.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+
+When users click a toolbar [item](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Configuration/toolbar/items/), it calls the method with the same name. You can also call these methods programmatically via the [getModules](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Methods/#getModulemoduleName) method. Refer to the following documentation file to see the full list of supported operations with examples: <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md">DevExtreme Quill modules: Table</a>.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/HtmlEditor/Tables/description.md]: > it calls the method with the same name

Passive voice seems more suitable here: "a method with the same name is called".

> call these methods **programmatically**

Looks exessive.

>  via the [getModules](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Methods/#getModulemoduleName) method

This is a bit vague. How do methods get called _via_ another method? We should give more details:

"These methods belong to the Table module of Quill. You can use the getModule method to access it and call Table methods at any point in your application."

> the full list of supported operations

"Table methods"?

File: JSDemos/Demos/HtmlEditor/Tables/description.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
+DevExtreme **HtmlEditor** allows users to create and manage tables. To enable this functionality, add the following items to the [toolbar](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Configuration/toolbar/):
```
Comment (RomanTsukanov) [inline JSDemos/Demos/HtmlEditor/Tables/description.md]:
```suggestion
DevExtreme **HtmlEditor** allows users to create and manage tables. To enable this functionality, add the following [items](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Configuration/toolbar/items/) to the [toolbar](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Configuration/toolbar/):
```

File: JSDemos/Demos/HtmlEditor/Tables/description.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deletetable" target="_blank">deleteTable</a>
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deleterow" target="_blank">deleteRow</a> / <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deletecolumn" target="_blank">deleteColumn</a>
+
+When users click a toolbar [item](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Configuration/toolbar/items/), a method with the same name is called. You can also call these methods via the  method. 
+
+These methods belong to the Table module of Quill. You can use the [getModules](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Methods/#getModulemoduleName) method to access this module and call its methods at any point in your application. Refer to the following documentation file to see the full list of the Table methods with examples: <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md">DevExtreme Quill modules: Table</a>.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/HtmlEditor/Tables/description.md]:
```suggestion
When users click a toolbar item, a method with the same name is called. These methods belong to the Table module of Quill. You can use the [getModules](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Methods/#getModulemoduleName) method to access this module and call its methods at any point in your application. Refer to the following documentation file to see the full list of the Table methods with examples: <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md">DevExtreme Quill modules: Table</a>.
```

**Review by RomanTsukanov — DISMISSED** [review]

File: JSDemos/Demos/HtmlEditor/Tables/description.md [hunk 9]
```diff
@@ -0,0 +1,9 @@
...
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deletetable" target="_blank">deleteTable</a>
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deleterow" target="_blank">deleteRow</a> / <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deletecolumn" target="_blank">deleteColumn</a>
+
+When users click a toolbar item, a method with the same name is called. These methods belong to the Table module of Quill. You can use the [getModules](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Methods/#getModulemoduleName) method to access this module and call its methods at any point in your application. Refer to the following documentation file to see the full list of the Table methods with examples: <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md">DevExtreme Quill modules: Table</a>.
```
Comment (Abadzhev) [inline JSDemos/Demos/HtmlEditor/Tables/description.md]: There's a lot of repetition of "modules" and "methods" so here's my compressed version - I think it ultimately says the same thing: 

```suggestion
You can initiate the same actions from code. Use the [getModules](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Methods/#getModulemoduleName) method to access the Table module. This module contains methods whose names match the options listed above. Refer to the following topic for the complete list and examples: <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md">DevExtreme Quill modules: Table</a>.
```

File: JSDemos/Demos/HtmlEditor/Tables/description.md [hunk 10]
```diff
@@ -0,0 +1,9 @@
...
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#insertcolumnright" target="_blank">insertColumnRight</a> / <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#insertcolumnleft" target="_blank">insertColumnLeft</a>
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deletetable" target="_blank">deleteTable</a>
+- <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deleterow" target="_blank">deleteRow</a> / <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md#deletecolumn" target="_blank">deleteColumn</a>
+
+You can initiate the same actions from code. Use the [getModules](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Methods/#getModulemoduleName) method to access the Table module. This module contains methods whose names match the options listed above. Refer to the following topic for the complete list and examples: <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md">DevExtreme Quill modules: Table</a>.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/HtmlEditor/Tables/description.md]:
```suggestion
You can initiate the same actions from code. Use the [getModules](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Methods/#getModulemoduleName) method to access the Table module. This module contains methods whose names match the toolbar item names listed above. Refer to the following topic for the complete list and examples: <a href="https://github.com/DevExpress/devextreme-quill/blob/master/docs/modules/table.md" target="_blank">DevExtreme Quill modules: Table</a>.
```

**Review by dirk-pieterse — DISMISSED** [review]

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**